### DISCUSSION TITLE
What is the release time of the stable v1.0.0?

### DISCUSSION THREAD

**Discussion by liudonghua123** [tauri-apps/tauri#3660] [General]
I would like to use official stable `tauri` in my app, so do you have a timeline or schedule for the v1.0.0.

**Comment by amrbashir** [comment]
There is no exact time frame but we are currently in the release candidate phase so it  should be soon.

**Reply by liudonghua123** [reply]
v1.0.0 has [released](https://github.com/tauri-apps/tauri/releases/tag/tauri-v1.0.0), and now I am looking forward to v2 which support more platforms like android and ios.
